public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json

```diff
@@ -48,7 +48,7 @@
       { "name": "Azure Monitor Network", "path": "img/azure_monitor_network.png" },
       { "name": "Azure Monitor CPU", "path": "img/azure_monitor_cpu.png" }
     ],
-    "version": "0.3.0",
+    "version": "1.0.0",
```

Comment:
this is an interesting theory! Should the version number match grafana's version number? I wonder if this should be automated or something like that when we release.

Reply:
Hmm, good point.
I noticed the other core plugins we own don't even have this version number. I feel like this is purely to override the external plugin's existing version, so it can be anything above 0.3.1. Not sure that making the version mirror the Grafana version would add value, though maybe I've missed something.
1.0.0 "feels" ok. Let me know if you have any other ideas!

Reply:
Starting to think we should clear the date under "updated"...

Reply:
+1 to delete the updated thing (or update it :sweat_smile: )

CloudMonitoring is also set to 1.0.0, so there is a "precedent"